tests_and_analysis/test/script_tests/test_powder_map.py

```diff
@@ -112,16 +112,27 @@ def test_invalid_weights_raises_causes_exit(self, powder_map_args):
 @patch('matplotlib.pyplot.show')
 @pytest.mark.skip(reason='Only run if you want to regenerate the test data')
 def test_regenerate_powder_map_data(_):
```

Comment:
I notice that the powder-map tests are being tweaked, but the adaptive broadening has not been added to this CLI tool. I think we should allow the option, even if it is tediously slow to run? (The test case can still be very small.)

Reply:
Yep the main reason I didn't add it was because I thought it'd be slow, but I'll give it a go

Reply:
If that turns out to be messy it could become a separate PR?

Reply:
Yeah, since the DOS could change slightly in #116, maybe easier to leave it until after. Opened PR #139